ui/raidboss/data/05-shb/raid/e4s.js

```diff
@@ -60,50 +53,15 @@
       regexJa: Regexes.startsUsing({ id: '4116', source: 'タイタン' }),
       regexCn: Regexes.startsUsing({ id: '4116', source: '泰坦' }),
       regexKo: Regexes.startsUsing({ id: '4116', source: '타이탄' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            ja: '自分にタンクバスター',
-            cn: '死刑点名',
-            ko: '탱버 대상자',
-          };
-        }
-      },
-      // As this seems to usually seems to be invulned,
-      // don't make a big deal out of it.
-      infoText: function(data, matches) {
-        if (matches.target == data.me)
-          return;
-        if (data.role != 'tank' && data.role != 'healer')
-          return;
-
-        return {
-          en: 'Buster on ' + data.ShortName(matches.target),
-          de: 'Tankbuster auf ' + data.ShortName(matches.target),
-          fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-          ja: data.ShortName(matches.target) + 'にタンクバスター',
-          cn: '死刑点 ' + data.ShortName(matches.target),
-          ko: '"' + data.ShortName(matches.target) + '" 탱버',
-        };
-      },
+      response: Responses.tankBuster(),
```

Comment:
add a condition for being a tank or a healer here too.  Also, preserve the comment about not making a big deal.